Scripts/RemediationScripts/Remediate-ConfigureConditionalAccessPolicyForPIM.ps1

```diff
@@ -158,7 +162,7 @@ function Configure-ConditionalAccessPolicyForPIM
 
         .DESCRIPTION
         Remediates 'Azure_Subscription_Configure_Conditional_Access_for_PIM' Control.
-        Configure Conditional Access (CA) policy for 'Owner', 'Contributor' and 'User Access Administrator' in the Subscription.
+        Configure Conditional Access (CA) policy for critical roles in the Subscription.
```

Comment:
Configure Conditional Access (CA) policy for **Azure** roles in the Subscription.

Reply:
Changed from critical to 'Eligible' as suggested.